lib/gooddata/models/process.rb

```diff
@@ -209,6 +211,51 @@ def deploy_from_appstore(path, options = { :client => GoodData.client, :project
         process
       end
 
+      def deploy_component(options = { :client => GoodData.client, :project => GoodData.project })
+        client, project = GoodData.get_client_and_project(options)
+
+        deploy_name = options[:name]
+        fail ArgumentError, 'options[:name] can not be nil or empty!' if deploy_name.nil? || deploy_name.empty?
+
+        component_name = options[:component_name]
+        fail ArgumentError 'options[:component_name] can not be nil or empty!' if component_name.nil?
+
+        component_version = options[:component_version]
+        fail ArgumentError 'options[:component_version] can not be nil or empty!' if component_version.nil?
+
+        type = options[:type]
+        fail ArgumentError 'options[:type] can not be nil or empty!' if type.nil?
+
+        process_id = options[:process_id]
+
+        data = {
+          process: {
+            name: deploy_name,
+            type: type,
+            component: {
+              name: component_name,
+              version: component_version
+            }
+          }
+        }
+
+        data[:process][:component][:earlyAccessVersion] = options[:early_access_version] if options[:early_access_version]
+        data[:process][:component][:configLocation] = options[:config_location] if options[:config_location]
+
+        res =
```

Comment:
Identical code found in process.rb:122, process.rb:196, process.rb:245 (mass = 198)